docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -0,0 +1,82 @@
+---
+title: C# Arrays | A tour of the C# language
```

Comment:
That's an interesting problem.  I think with F# it's a bit easier since arrays are different (although they still have almost the same semantics), but it's still so many shared concepts.

I think it's best if we accept that there will be duplicated efforts here and cross-link.  The .NET arrays topic should link to the C# and F# tours.  Perhaps they should also link to the .NET arrays topic, even if it's mostly the same info.  That broadens search and gets us more hits.